README.md

```diff
@@ -1,2 +1,94 @@
 # variant-remapping
-The pipeline for remapping VCF variants between two arbitrary FASTA assemblies.
+Pipeline for remapping VCF variants between two arbitrary assemblies in FASTA format. No chain file is required.
+
+**Method**: creates reads from the flanking sequences of each variant, then maps them to the new assembly using 
+bowtie2.
+
+Currently, it only supports SNPs, and not indels.
+
+**Prerequisites:**
+- reverse_strand.py, included (uses pysam, allows reverse strand allele correction (see Note below for further 
+explanation))
+- [vcf2bed](https://bedops.readthedocs.io/en/latest/content/reference/file-management/conversion/vcf2bed.html)
+- [bowtie2](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml)
+- [samtools](http://www.htslib.org/download/)
+- [bedtools](https://bedtools.readthedocs.io/en/latest/)
+- [bcftools](http://www.htslib.org/download/)
+
+**Note about the reverse strand correction:**  
+When a variant is mapped to the reverse strand, the corresponding allele in the output VCF file is reversed, aka 
+converted to the forward strand, as alleles in VCF files are always described on the forward strand. For example:
+INPUT VCF:
+Old genome: G (ref allele) --> A (variant allele)
+This maps onto the new genome on the reverse strand:
+OUTPUT VCF:
+New genome: C (ref allele) --> T (variant allele)
+
+## Input
+- Old genome assembly file (FASTA format): the genome you have variants for.
+- New genome assembly file (FASTA format): the genome you want to remap the variants to.
+- Variants file (VCF format): contains the list of variants you want to remap.
+- Accession name for the new assembly: required to recreate the header for the output VCF file.
+
+## Output
+A VCF file containing:
+- remapped coordinates (position on the new assembly)
+- rsIDs
+- the correct chromosome/contig names
+- the new REF alleles (reverse strand mapping taken into account)
+- the ALT, QUAL, FILT and INFO columns of the input VCF
+
+Console output: 
+```
+"Total number of input variants:"
+[number]
+"Total number of variants processed (after filters):"
+[number]
+"Number of remapped variants:"
+[number]
+"Percentage of remapped variants:"
+[percentage]
+```
+
+## Usage
+First, make sure the scripts are executable:
+`chmod a+x remapping_commands.sh reverse_strand.py`
+
+Command usage: (see Input for details)
+`./remapping_commands.sh -g [oldgenome] -n [newgenome] -a [newgenomeaccession] -v [vcffile] -o [outfile name]`
+
+You can use `time` at the beginning of the previous command to get the runtime and CPU time.
+
+**Example command:**
+`time ./remapping_commands.sh \
+  -g droso_dm3.fasta \
+  -n droso_dm6.fasta \
+  -a GCA_000001215.4 \
+  -v droso_variants_renamed.vcf \
+  -o test.vcf`
```

Comment:
I don't think this will properly format because single backticks are only meant to be used for inline code formatting. Here it's better to use block code formatting with triple backticks (as shown here https://help.github.com/en/articles/creating-and-highlighting-code-blocks)